topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.md

```diff
@@ -158,31 +160,33 @@ This first step consist of downloading and properly prepare the data to use it i
 >
 >    {% include snippets/add_tag.md %}
 >
-> 5. Change datatype CSV to tabular for each data file
+>    If tags doesn't display directly on your history, refresh the page.
+>
+> 5. Convert datatype CSV to tabular for each data file
 >
->    {% include snippets/change_datatype.md datatype="datatypes" %}
+>    {% include snippets/convert_datatype.md conversion="Convert CSV to tabular" %}
 >
 {: .hands_on}
 
 These three data files gather Catch Per Unit Effort (CPUE) observations, a relative abundance measure mostly used 
 to assess exploited fish species populations :
- - EVHOE {% cite EVHOE %} is a French survey on Southern Atlantic Bottom Trawl, the extracted file available on Zenodo goes from 
+ - EVHOE ({% cite EVHOE %}) is a French survey on Southern Atlantic Bottom Trawl, the extracted file available on Zenodo goes from 
    1997 to 2018 and records CPUE for seven exploited fish species
- - BITS {% cite BITS %} is an international survey on Trawl in Baltic sea, the extracted file available on Zenodo goes from 1991
+ - BITS ({% cite BITS %}) is an international survey on Trawl in Baltic sea, the extracted file available on Zenodo goes from 1991
    to 2020 and records CPUE for three exploited fish species
- - SWC-IBTS {% cite SWCIBTS %} is a Scottish survey on bottom trawl in the west coast, the extracted file available on Zenodo goes 
+ - SWC-IBTS ({% cite SWCIBTS %}) is a Scottish survey on bottom trawl in the west coast, the extracted file available on Zenodo goes 
    from 1985 to 2010 and records CPUE for six exploited fish species
 
 ## Prepare the data
 
 Before starting the preparation of the data, we need to identify which inputs we need for the tools and their format
 (same for Community or Species-Population analyses except for a few details): 
- - Input for the tools to compute metrics: Observation data file containing at least ```year``` and "location" sampled 
+ - Input for the tools to compute metrics: Observation data file containing at least ```'year'``` and "location" sampled 
    that might be represented in one field "observation.unit", "species.code" to name the species sampled, 
-   and ```number``` as the abundance value
+   and ```'number'``` as the abundance value
```

Comment:
not important, but just and FYI for the future, it should be the same if you just use single backticks instead of triple ones

Reply:
Oh noted ! Thank you very much :)

Reply:
Thank you Saskia !!!